### PR TITLE
Split features count and grid loading, update geoserver url to sira2

### PR DESCRIPTION
**Pull request by kappu72**

### CODE REVIEW

File: aua.properties [hunk 1]
```diff
@@ -6,36 +6,36 @@ fields=prov,com,tipo_rec,tipo_rif,op_rec,data_provv
 prov.id=sira:impianto/sira:Sede/sira:provincia
 prov.name=Provincia
 prov.type=list
-prov.valueService=http://sira.csi.geo-solutions.it/geoserver/ows?service=WFS&version=1.1.0&request=GetFeature&typeName=sira:province&outputFormat=application%2Fjson
+prov.valueService=http://sira2.csi.geo-solutions.it/geoserver/ows?service=WFS&version=1.1.0&request=GetFeature&typeName=sira:province&outputFormat=application%2Fjson
```
Comment (mbarto) [inline aua.properties]: I don't think this file is used anymore

File: js/components/template/PreviewMap.jsx [hunk 2]
```diff
@@ -113,7 +113,7 @@ module.exports = connect((state) => {
             },
             {
               "type": "wms",
-              "url": "http://sira.csi.geo-solutions.it/geoserver/wms",
+              "url": "http://sira2.csi.geo-solutions.it/geoserver/wms",
```
Comment (mbarto) [inline js/components/template/PreviewMap.jsx]: this should be parametrized

File: aua.properties (hunk 1 above)
Comment (kappu72) [inline aua.properties]: I'll remove

File: js/components/template/PreviewMap.jsx (hunk 2 above)
Comment (kappu72) [inline js/components/template/PreviewMap.jsx]: Ok

**Pull request merged**